genie-web/src/integTest/java/com/netflix/genie/web/apis/rest/v3/controllers/JobRestControllerAgentExecutionIntegrationTest.java

```diff
@@ -28,7 +29,8 @@
  */
 @TestPropertySource(
     properties = {
-        JobExecutionModeSelector.DEFAULT_EXECUTION_PROPERTY + "=true"
+        JobExecutionModeSelector.DEFAULT_EXECUTION_PROPERTY + "=true",
+        LocalAgentLauncherProperties.PROPERTY_PREFIX + ".run-as-user=false",
```

Comment:
Not that we may want to since we're going to remove it (hopefully) but just pointing out that this property isn't documented